src/modules/url-preview/getContentType.js

```diff
@@ -0,0 +1,14 @@
+import 'isomorphic-fetch';
+
+export default async function(url: string): Promise<string> {
+	return new Promise((resolve, reject) => {
```

Comment:
No need to create a new promise. Just return the promise by fetch. You can return inside then instead of resolve and throw error instead of reject.